src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
#1604 was a thread on practices such as this a while back.  There are some good points made, such as using a public API interface for testing means the tests are portable among different implementations of the framework.  In addition, the fact that we need the internals visible in order to test them implies that the public interface is not able to exercise them properly, which in turn implies there is dead code somewhere.